java/src/main/native/src/TableJni.cpp

```diff
@@ -887,6 +888,7 @@ JNIEXPORT long JNICALL Java_ai_rapids_cudf_Table_writeParquetBufferBegin(
         .nullable_metadata(&metadata)
         .compression(static_cast<compression_type>(j_compression))
         .stats_level(static_cast<statistics_freq>(j_stats_freq))
+        //.int96_timestamps(true)
```

Comment:
please commencement this and set it to the correct value.  the dependency has been merged in.